api/routers/report.py

```diff
@@ -102,37 +111,107 @@ async def update_reports(old_user_id: int, new_user_id: int, token: str):
 
 
 @router.post("/v1/report", tags=["Report"])
-async def insert_report(token: str, detections: List[detection]):
+async def insert_report(request : Request,
+    detections: List[detections],
+    manual_detect: int = Query(0, ge=0, le=1),
+    ):
     '''
-        Work in progress 
-        Insert report.
+        Inserts detections to the plugin database.
     '''
-    await verify_token(token, verification='verify_ban', route='[POST]/v1/report/')
+    client_host = request.client.host
+    success = JSONResponse(status_code=status.HTTP_200_OK, content=jsonable_encoder({'OK': 'OK'}))
+    bad_data = JSONResponse(status_code=status.HTTP_400_BAD_REQUEST, content=jsonable_encoder({'error': 'There was an error with your request. Please contact plugin support.'}))
```

Comment:
don't think we need =jsonable_encoder(), a dict should work

Reply:
Either works, I'll review using dict only.